lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -505,4 +745,34 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
       {Enum.reverse(segments_acc), Enum.reverse(headers_acc), segments, window_duration, header}
     end
   end
+
+  defp creation_time(:live), do: [{:creation_time, DateTime.utc_now()}]
+  defp creation_time(:vod), do: []
+
+  defp get_partial_sequence_number(
```

Comment:
Would be more readable if you've merged that into a single function and used case/cond